qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -29,6 +29,13 @@
                     no RHEL.6
                     cpu_model_flags += ",+avx512vbmi2,+avx512vnni,+gfni,+vaes,+avx512bitalg,+vpclmulqdq"
                     flags = "avx512_vbmi2 avx512_vnni gfni vaes avx512_bitalg vpclmulqdq"
+                - arch_capabilities:
+                    # support RHEL.7 guest or later
+                    no RHEL.6
+                    cpu_model_flags += ",arch-capabilities=on,+mds-no"
+                    flags = "arch_capabilities"
+                    check_guest_cmd = "cat /sys/devices/system/cpu/vulnerabilities/mds | grep '%s'"
+                    items = 'Not affected'
```

Comment:
How about name it `expect_result` or `expect_items`?

Reply:
thanks and update.